api/entity_config/config.go

```diff
@@ -12,6 +12,12 @@ func ConfigValue(network *net.Network, application, entity, config string) (stri
 	if nil != err {
 		return "", err
 	}
+
+	var value string;
+	err = json.Unmarshal(bytes, &value)
+	if nil == err {
```

Comment:
But then the calling code will see it as a failure - 
```go
config, err := entity_config.ConfigValue(cmd.network, scope.Application, scope.Entity, c.Args().First())
        if nil != err {
            fmt.Fprintln(os.Stderr, err)
            os.Exit(1)
        }
```

whereas the intent of the code in ConfigValue is to:
1. Try interpreting it as a single JSON string (line 17); if this works, return the string value (line 19).
2. If 1. doesn't work, it is some more complex JSON expression, so just return the literal JSON (line 21).

A failure to unmarshal at line 17 is fine, it just means the value isn't a simple "json string".